modern-normalize.css

```diff
@@ -204,13 +201,18 @@ button::-moz-focus-inner,
 Restore the focus styles unset by the previous rule.
 */
 
-button:-moz-focusring,
-[type='button']:-moz-focusring,
-[type='reset']:-moz-focusring,
-[type='submit']:-moz-focusring {
+:-moz-focusring {
 	outline: 1px dotted ButtonText;
 }
 
+/**
+Remove the additional ':invalid' styles in Firefox.
```

Comment:
Are you sure this is still valid? Can you try to find a Firefox bug for this a link to it in the comment?

Reply:
I've checked the source code of Firefox and can confirm that the `box-shadow` property is [overridden through UA sheets](https://dxr.mozilla.org/mozilla-central/rev/c68fe15a81fc2dc9fc5765f3be2573519c09b6c1/layout/style/res/forms.css#745-754).

More code search results can be found [here](https://dxr.mozilla.org/mozilla-central/search?q=%22%3A-moz-ui-invalid%22&redirect=false).

Reply:
Can you add a link to it in the comment? Using this mirror https://github.com/mozilla/gecko-dev

Reply:
Sure, I've just added it.